MikelSeguraElizalde/p1/1_phonetics.ipynb

```diff
@@ -0,0 +1,855 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.            if ipa_query == "NOT FOUND":
</pre><p>Hubiera estado bueno modularizar un poco más con funciones para no tener todo en el <code>while</code> . Esto hace que el código sea algo difícil de leer. Por ejemplo, aquí  en todo lo que sigue de este <code>if</code> podría entrar en  una función llamada<code>get_similar_words()</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/umoqnier/cl-2024-1-lab/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>